.github/actions/cdn_code_build/action.yaml

```diff
@@ -0,0 +1,87 @@
+name: NodeJS CDN Code Build 
```

Comment:
it looks like this is a generic build action for yarn based project (being them websites or backends). the only peculiar part is the "Define env-config.js" step, which, imho should be delegated to the project build task someway (though npm tasks). 

I'm not convinced that this action adds value, what do you think @lucacavallaro ?

Reply:
I agree. Usually the ""environment variables"" (in frontend there aren't proper env variables 😛) are managed by the framework - in Next project for example we can commit different `.env.SOMETHING` and pass them as build argument.

Reply:
So do you think it's better to remove this specificity and make the `code_build` part generic?

Reply:
yep (and we can share the `code build` part between backend and frontend projects)

Reply:
maybe we can think about this in a second phase